README.md

```
@@ -29,7 +29,7 @@ git pull --recurse-submodules
 ```
 
 For more information, please refer to the [git
-documentation](http://git-scm.com/).
+docum[XCSoar-debug.apk](output/ANDROID/bin/XCSoar-debug.apk)entation](http://git-scm.com/).
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)